src/eva/utilities/utils.py

```diff
@@ -336,3 +336,13 @@ def slice_var_from_str(config, datavar, logger):
 
 
 # --------------------------------------------------------------------------------------------------
+
+
+def is_number(s):
```

Comment:
does this exist already in python somewhere maybe? or not?

Reply:
Seems like it should but I haven't found it so far.  This solution was a cut and paste from stackoverflow.

Reply:
works for me, just wanted to check